pack8s/cmd/run.go

```diff
@@ -24,25 +24,32 @@ import (
 )
 
 type runOptions struct {
-	privileged     bool
```

Comment:
pack8s is not really maintained or used. `gocli` now also works with latest `podman` and it s docker-api support. We should remove `pack8s`.

Reply:
Humm, ok, I will revert all changes in pack8s